source/fundamentals/connection/tls.txt

```diff
@@ -37,6 +37,17 @@ You can enable TLS/SSL for the connection to your MongoDB instance
 in two different ways: through a parameter in your connection string, or
 using a method in the ``MongoClientSettings.Builder`` class.
 
+.. note::
+
+   If you connect by using the DNS seedlist protocol, indicated by the ``+srv``
+   connection string modifier, the driver enables TLS/SSL. To disable it,
```

Comment:
S: [Compound adjective](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#hyphens-in-compound-modifiers)

```suggestion
   connection-string modifier, the driver enables TLS/SSL. To disable it,
```

Reply:
Thanks for the suggestion! I had intended to keep it consistent with the Server manual mention (linked in the PR text), but I think it could be more clear to describe a little differently.